test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -0,0 +1,725 @@
+        this.setState({platformFilter: platformFilter});
+        this.populateTable(info, type);
+        this.populateCompTable();
+    }
+
+    render() {
+        const tProps = {
+            value: this.state.benchmarkFilter,
+            onChange: this.onBenchmarkChange,
+            treeCheckable: true,
+            showCheckedStrategy: SHOW_PARENT,
+            searchPlaceholder: "Please choose the benchmarks to view",
+            style: {
+                 width: 1000
+            }
+        };
+        return (
+
+            <div>
+                <div className="row">
+                    <div className="column" style={{fontSize:20, fontWeight:'bold'}}> Parameters <br/> </div>
+                    <div className="column" style={{fontSize:20, fontWeight:'bold'}}> Test <br/> </div>
+                    <div className="column" style={{fontSize:20, fontWeight:'bold'}}> Baseline <br/> </div>
+            </div>
+            <div className="row">
+                    <div className="column" style={colStyle}> JDK Date <br/> </div>
+                    <div className="column"> 
+                        <DayPickerInput
+                            onDayChange={this.handleDayChange}
+                            value={this.state.testJdkDate}
+                            overlayComponent={CustomOverlay}
+                            dayPickerProps={{
+                                todayButton: 'Today',
+                                type: 'test'
+                            }}
+                            keepFocus={false}
+                        /> 
+            <Tooltip placement="topRight" title="Table will contain latest results from all builds dated before the chosen date regardless of when the benchmark was run.">
+                <Icon type="question-circle" />
+            </Tooltip></div>
+            <div className="column"> 
+                <DayPickerInput
+                    onDayChange={this.handleDayChange}
+                    value={this.state.baselineJdkDate}
+                    overlayComponent={CustomOverlay}
+                    dayPickerProps={{
+                        todayButton: 'Today',
+                        type: 'baseline'
+                    }}
+                    keepFocus={false}
+                /> 
+            <Tooltip placement="topRight" title="Table will contain latest results from all builds dated before the chosen date regardless of when the benchmark was run.">
+                <Icon type="question-circle" />
+            </Tooltip></div>
+        </div>
+        <div className="row">
+            <div className="column" style={colStyle}> JDK Version <br/> </div>
+            <div className="column"> <select name="testJdkVersion" className="select-css" value={this.state.testJdkVersion} onChange={this.handleChange.bind(this)}>
+                <option value="O8">OpenJDK8</option>
+                <option value="O11">OpenJDK11</option>
+                <option value="O12">OpenJDK12</option>
```

Comment:
We should not use the dropdown and hardcode jdk versions. It is a lot of work to update this for every release. An input box with a default value is a better choice. Or if you want to use dropdown, then the data should come from the database.

Same for the rest of dropdown. And each of them should be a react component. 
If you do not have time to fix it now, please open an issue for it.

Reply:
Yeah this one definitely makes sense. Else, we'll have to update these options whenever there's a new java version or type.

Reply:
@llxia Soft-coding these options would add more `distinct` in the code. Are you fine with the more additions or do we leave this hard-coded for now? Thanks!

Reply:
Nvm! We found a more efficient way to do it by grouping with `$addToSet`, so we are soft-coding all those options.